homeassistant/components/lutron_caseta/binary_sensor.py

```diff
@@ -65,16 +85,6 @@ def unique_id(self):
         """Return a unique identifier."""
         return f"occupancygroup_{self.device_id}"
```

Comment:
This is not actually unique if you have multiple bridges. We need to prefix it with the bridge id as well since it is unique to the bridge.

Reply:
We can migrate them in 2022.6 though as its better to fix this now instead of adding risk